indico/modules/events/tracks/blueprint.py

```diff
@@ -16,13 +16,17 @@
 
 from __future__ import unicode_literals
 
-from indico.modules.events.tracks.controllers import RHManageTracks, RHCreateTrack, RHCreateTrackOld
+from indico.modules.events.tracks.controllers import (RHManageTracks, RHCreateTrack, RHCreateTrackOld, RHEditTrack,
+                                                      RHSortTracks, RHDeleteTrack)
 from indico.web.flask.wrappers import IndicoBlueprint
 
 _bp = IndicoBlueprint('tracks', __name__, template_folder='templates', virtual_template_folder='events/tracks',
-                      url_prefix='/event/<confId>')
+                      url_prefix='/event/<confId>/manage')
```

Comment:
#2651 adds `/program` which is not management. So I'd keep the `/manage` explicit instead of adding it to the prefix.